docker-compose.yml

```diff
@@ -0,0 +1,31 @@
+version: "3.8"
+
+services:
+  mongodb:
+    image: mongo:5.0.2
+    restart: unless-stopped
+    env_file: ./.env
+    ports:
+      - 27017:27017
+    volumes:
+      - ./mongorestore.sh:/docker-entrypoint-initdb.d/mongorestore.sh
+      - ./test/data:/db-dump
+    networks:
+      - rick-and-morty-api-net
+  app:
+    depends_on:
+      - mongodb
+    image: rick-and-morty-api:latest
+    restart: unless-stopped
+    build:
+      dockerfile: Dockerfile
+    environment:
+      - NODE_ENV=production
```

Comment:
```
- NODE_ENV=production
```

I'm not sure if I agree with this, the environment is not `production`.  I understand that you probably added this so that the [API uses the correct db](https://github.com/afuh/rick-and-morty-api/blob/master/server.js#L20-L21).

`const db = process.env.NODE_ENV === 'production' ? process.env.DATABASE : LOCAL`

However, perhaps we should update [`server.js`](https://github.com/afuh/rick-and-morty-api/blob/master/server.js) to be more consistent and only use `DATABASE` as env. variable.